### DISCUSSION TITLE
Normal slope objects

### DISCUSSION THREAD

**Discussion by mariomadproductions** [city41/smaghetti#176] [General]
These are the results of some testing I did with hex tree and the e-reader level Slidin' the Slopes. The red square represents the object position. This is incomplete of course, I'll probably do some more testing later.

| Bank | ID | Param1 | Param2 | Image | Notes
|---|---|---|---|---|---|
| 4 | 31| 0 | N/A | ![image](https://user-images.githubusercontent.com/9919020/175841953-96f7e496-add1-472f-a992-eb6039ef3dfa.png)<br>![image](https://user-images.githubusercontent.com/9919020/175843392-a25d6c20-d2d8-4fde-b1d9-c6ab38a53968.png)
| 4 | 32 | 0 | N/A | ![image](https://user-images.githubusercontent.com/9919020/175842626-0faf5bfb-01f9-4aff-9193-a35bced8dcbe.png)<br>![image](https://user-images.githubusercontent.com/9919020/175842873-f7f4c7be-01fb-4485-ad79-f45b4191973f.png)
| 4 | 32 | 1 | N/A | ![image](https://user-images.githubusercontent.com/9919020/175843584-93622bdc-3770-461c-99df-1fa27ec83c5e.png)<br>![image](https://user-images.githubusercontent.com/9919020/175843588-94cd971c-5ed7-4916-afb1-bc51d3f33668.png)
| 4 | 32 | 2 | N/A | ![image](https://user-images.githubusercontent.com/9919020/175844123-10f13f6a-baec-4d23-ab33-bc783509c846.png)<br>![image](https://user-images.githubusercontent.com/9919020/175844134-a191fa8b-08dd-4712-95ec-153e8c2b37f9.png)
| 4 | 74 | 1 | 00 | ![image](https://user-images.githubusercontent.com/9919020/175845841-d7971a00-8e37-4d61-9b4f-42a4e7317d88.png)<br>![image](https://user-images.githubusercontent.com/9919020/175845854-63bc1e6d-15a0-403f-8bba-a7d1743ba0ff.png) | 
| 4 | 75 | 1 | 00 | ![image](https://user-images.githubusercontent.com/9919020/175844711-a92c1196-b707-4c50-80cd-f41d3aae93eb.png)<br>![image](https://user-images.githubusercontent.com/9919020/175844725-a4c2cb81-fc71-4d64-8f70-2be07a802f85.png) | Param1=0 makes the slope use glitched graphics, in this level at least
| 4 | 75 | 1 | 01 | ![image](https://user-images.githubusercontent.com/9919020/175845533-8e895ae2-71e8-45c4-b7bb-4ed8f61c440e.png)<br>![image](https://user-images.githubusercontent.com/9919020/175845539-b60d8e1b-629e-48b6-89f9-d45aaa53bf9b.png) |

**Comment by city41** [comment]
Nice, thanks. I created #177 to track these getting into the editor.

**Reply by mariomadproductions** [reply]
Thanks!
